addons/core/translations/resources/en-us.yaml

```
@@ -925,10 +925,32 @@ policy:
     delete_after: Delete After
     do_not_delete: Do not delete
     forever: Forever
+    days: days
   actions:
     delete: Delete Storage Policy
   messages:
     none:
       title: No storage policies yet
       description: Storage policies define how long to store recordings before they are automatically deleted. We recommend checking your data retention policies for compliance requirements.
       link: Create a new storage policy
+  form:
+    retention_policy:
+      label: Retention Policy
+      help: Specifies how long a recording must be stored
```

Small, nitpick is that the help translations usually end with a period (.)
```suggestion
      help: Specifies how long a recording must be stored.
```